infrastructure/scripts/conf.sh

```diff
@@ -5,30 +5,35 @@ IFS=$'\n\t'
 SCRIPT_PATH=$(cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P)
 INFRASTRUCTURE_PATH=$(cd ${SCRIPT_PATH}/../; pwd -P)
 
-if [[ ! -f ${INFRASTRUCTURE_PATH}/airy.conf ]]; then
-    echo "No airy.conf config file found"
+if [[ ! -f ${INFRASTRUCTURE_PATH}/airy.yaml ]]; then
+    echo "No airy.yaml config file found"
     exit 0
 fi
 
-source ${INFRASTRUCTURE_PATH}/scripts/lib/k8s.sh
-
-DEPLOYED_AIRY_VERSION=`kubectl get configmap core-config -o jsonpath='{.data.APP_IMAGE_TAG}'`
-if $(grep -q "  appImageTag" ${INFRASTRUCTURE_PATH}/airy.conf); then
-    CONFIGURED_AIRY_VERSION=$(grep "  appImageTag" ${INFRASTRUCTURE_PATH}/airy.conf | head -n 1 | awk '{ print $2}')
-else
-    CONFIGURED_AIRY_VERSION=""
+source /vagrant/scripts/lib/k8s.sh
+
+if [ -z ${AIRY_VERSION+x} ]; then
+    branch_name="$(git symbolic-ref HEAD 2>/dev/null)" ||
+    branch_name="(unnamed branch)"     # detached HEAD
+
+    branch_name=${branch_name##refs/heads/}
+    case "$branch_name" in
+        main|release* )
+            AIRY_VERSION=(`cat ../../VERSION`)
+            ;;
+        * )
+            AIRY_VERSION=develop
+            ;;
+    esac
 fi
 
-if [ -z ${CONFIGURED_AIRY_VERSION} ]; then
-    AIRY_VERSION=${DEPLOYED_AIRY_VERSION}
-else
-    AIRY_VERSION=${CONFIGURED_AIRY_VERSION}
-fi
+source ${INFRASTRUCTURE_PATH}/scripts/lib/k8s.sh
+
 
 kubectl delete pod startup-helper --force 2>/dev/null || true
 kubectl run startup-helper --image busybox --command -- /bin/sh -c "tail -f /dev/null"
 
-helm upgrade core ${INFRASTRUCTURE_PATH}/helm-chart/ --values ${INFRASTRUCTURE_PATH}/airy.conf --set global.appImageTag=${AIRY_VERSION} --timeout 1000s > /dev/null 2>&1
+helm upgrade core ${INFRASTRUCTURE_PATH}/helm-chart/ --values ${INFRASTRUCTURE_PATH}/airy.yaml --set global.appImageTag=${AIRY_VERSION} --timeout 1000s > /dev/null 2>&1
 
 kubectl scale deployment schema-registry --replicas=1
 
```

Comment:
This way, we don't consider the version with which the core was deployed, as well as if we are on `main` and we want to `override` the version.
Or if we are on another branch and want to test a specific version (example 0.5.0).

Reply:
you can always export AIRY_VERSION=beta before starting the bootstrap